dev_guide/builds.adoc

```diff
@@ -119,20 +187,20 @@ code. If your application exists inside a sub-directory, you can override the
 default location (the root folder) using this field.
 ====
 
-[[configuring-the-sti-environment]]
-== STI Environment
+[[configuring-the-source-environment]]
+== Source Environment
 
-There are two ways to make environment variables available to the link:../architecture/core_objects/builds.html#sti-build[STI] 
+There are two ways to make environment variables available to the link:../architecture/core_objects/builds.html#source-build[Source] 
 build process and resulting image, Environment files and BuildConfig environment values.  
 
 === Environment files
-STI enables you to set environment values
+Source enables you to set environment values
 in your application by specifying them in a *_.sti/environment_* file in the
 source repository. The environment variables are then present during the build
 process and in the final docker image. The complete list of supported
 environment variables are available in the documentation for each image.
 
-If you provide a *_.sti/environment_* file in your source repository, STI reads
+If you provide a *_.sti/environment_* file in your source repository, Source reads
```

Comment:
Source build or S2I or something.  Source doesn't make sense in this context.  Victim of a search and replace i suppose.